src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js

```diff
@@ -191,13 +194,31 @@ export const AzureComputeModalBase = ({
     )
   }
 
+  const sendCloudEnvironmentMetrics = () => {
+    const metricsEvent = Utils.cond(
+      [(viewMode === 'deleteEnvironment'), () => 'cloudEnvironmentDelete'],
+      //[(!!existingRuntime), () => 'cloudEnvironmentUpdate'], TODO: When update is available, include in metrics
```

Comment:
Would it make sense to create a ticket in the backlog summarizing the updates needed and link here?

Reply:
Yes, I'll get on that - I'll make the ticket strictly for mixpanel metrics and we can link it to the update feature ticket, which I'm unsure if it exists